praw/models/reddit/mixins/__init__.py

```diff
@@ -112,6 +115,40 @@ def remove(self, spam=False):
         data = {'id': self.thing.fullname, 'spam': bool(spam)}
         self.thing._reddit.post(API_PATH['remove'], data=data)
 
+    def send_removal_message(self, type,  # pylint: disable=redefined-builtin
+                             title, message):
```

Comment:
Any chance you would want to switch the order of `title` and `message`, and add a default value to `title`? I ask because I think it might be a little awkward to call something like `post.mod.send_removal_message('public', None, 'My message here')`. It could be nicer if the method signature was `def send_removal_message(self, type, message, title='')`

Reply:
Actually, it seems to 400 when that field is `None` or `''`. Any idea why, since as you say in the docstring, that field is ignored for `'public'` removal messages?

Reply:
The official client always uses removal messages and removal reasons at the same time.  The user has to choose a predefined removal reason, and the value for `title` is taken from there, without any opportunity to edit it.  I guess there's always expected to be a title, even if it's unused.

Reply:
How do you feel about moving `type` to the last argument, make its default be public, and then move `title` to the second to last argument with it's default as `'ignored'`.